### PR TITLE
chore: update backend sample.env

### LINKED ISSUE #393: Update sample.env for backend
### Current Behavior
Backend sample.env is not updated

### Expected behavior/code
Sample to have env vars for all cases

### ⚙️ Environment

- `Python` version(s): All
- `OS`: All

### Possible Solution

Have sample.env look something like this:
```
# required
MONGO_URI=......
....
...
# deployment 
FLASK_URI=....
...
..
# upload to github
GITHUB_ID=...
....
...
```

### Additional context/Screenshots
Ppl will know the bare minimum needed to run the application and can add stuff as they want them :D

### PR DESCRIPTION
**Pull request by McTechie**
# Pull Request

## What does this PR do?
Fixes #393

Added bare minimum comments to guide users in `BackEndApp/sample.env`

## What part does this affect?
- [ ] FrontEnd.
- [x] BackEnd.
- [ ] Documentation.
- [x] Other. (Please specify below)

Sample Env file present in `BackEndApp`

## Before submitting
- [x] Was this discussed/approved via a GitHub issue or slack?
- [x] Did you read the [contributor guideline](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md)?
- [x] Did you ensure that there aren't any other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change?
- [x] Did you make sure the title is self-explanatory and the description concisely explains the PR?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure the code is clean and docstrings have been added or updated as required?
- [x] Did you make sure the code is linted/formatted locally prior to submission? (using `black` and/or `prettier`)
- [ ] Did you make sure to update the documentation with your changes? (if necessary)

## PR review
Anyone in the community is free to review the PR once the tests have passed.

Thank you for contributing to [AutoDL](https://github.com/Auto-DL/Auto-DL). We look forward to your continued support.

### CODE REVIEW

**Comment by bamblebam** [comment]
Is it alright for these things to be public ?

**Comment by McTechie** [comment]
Ahh yes actually urls won't be problem, and as for `jwt_secret` and `github` tokens (present previously as well)... I'm sure that these are dummy values (or at least something which wouldn't matter being public). xD

@ADI10HERO thoughts on this?

File: BackEndApp/sample.env [hunk 1]
```diff
@@ -1,6 +1,15 @@
+# required
 MONGODB_URI=mongodb://localhost:27017/
 JWT_SECRET=abcdefh12345
+
+# model deployments
+FLASK_APP_HTTPS_URL=https://github.com/Auto-DL/Deployment-Flask.git
+FLASK_APP_SSH_URL=git@github.com:Auto-DL/Deployment-Flask.git
+
+# client app url
```
Comment (ADI10HERO) [inline BackEndApp/sample.env:9]: I think we can move this to required itself? 🤔  or add `(required)` in front of it..

**Comment by ADI10HERO** [comment]
> Ahh yes actually urls won't be problem, and as for `jwt_secret` and `github` tokens (present previously as well)... I'm sure that these are dummy values (or at least something which wouldn't matter being public). xD
> 
> @ADI10HERO thoughts on this?

Regarding this, I think we are fine!

File: BackEndApp/sample.env (hunk 1 above)
Comment (McTechie) [reply BackEndApp/sample.env]: I've moved it to required itself 👍🏼

**Review by ADI10HERO — APPROVED** [review]

**Pull request merged**